compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -311,6 +329,15 @@ OMR::ARM64::CodeGenerator::doBinaryEncoding()
       }
 
    self()->getLinkage()->performPostBinaryEncoding();
+
+   if (data.recomp != NULL && data.recomp->couldBeCompiledAgain())
+      {
+      J9::PrivateLinkage::LinkageInfo *lkInfo = J9::PrivateLinkage::LinkageInfo::get(self()->getCodeStart());
```

Comment:
You can't add J9-specific code in OMR.  Given that this code deals with private linkage, perhaps it should move into the OpenJ9 implementation of `performPostBinaryEncoding`, which is conveniently called just before this.

Reply:
Moved the code to `performPostBinaryEncoding()` in the OpenJ9 side.